product_docs/docs/pgd/5/consistency/conflicts/01_avoid_tolerate_conflicts.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Avoiding or tolerating conflicts
+Description: How to avoid database conflicts.
+deepToC: true
```

Comment:
Not necessary with no headings

Reply:
Also consider merging this section with the overview. It may make more sense together. I hadn't considered how tightly dependent they are.

Reply:
Moved as suggested.